### ISSUE TITLE
Project references files that are ignored by the VCS 

### ISSUE THREAD

**Issue by AngeloStavrow** [duckduckgo/ios-search-and-stories#150]
The files in question are the _ProximaNova_ fonts, which I assume cannot be included in this repository for licensing reasons.

And it's _such_ a great font! Unfortunately, this also means that Xcode builds will fail unless
1. the font is purchased by each contributor (USD$290 was the cheapest I found); or
2. the font is replaced by either something from the system, or an open-source font like [Open Sans](http://opensans.com).

Option 1 may not be financially viable for some members of the community, and option 2 may not be obvious to a lot of newcomers to the platform, as it's not mentioned in the setup section of [the contribution guidelines](https://github.com/duckduckgo/ios/blob/master/CONTRIBUTING.md), but it may become a headache to keep reverting these back to _ProximaNova_ for submission to the App Store.

I'll submit a PR for changes to `CONTRIBUTING.md` in the meanwhile.

**Comment by tfarina** [comment]
Since this was merged, can we close this?

**Comment by AngeloStavrow** [comment]
While the PR addresses some missing setup steps in the contribution guidelines, I think a second item that the community may want to address is whether or not it makes sense to drop the use of a licensed font in the project.

It don't know if there are practical considerations for using ProximaNova, including support for non-Latin characters. It _would_, however, make it easier a little to merge back PRs that aren't using a bunch of different fonts, as I mentioned above.

If so, I'm happy to submit a second PR to move the project over to either a system or open-source font.

**Comment by nilnilnil** [comment]
From my cursory understanding, we aren't wed to using custom fonts. @thm could speak more on this. It should be very easy to drop them if we don't think they'd add much to the application. As an aside, they're not a huge impedance either for people that want to make contributions.

**Comment by thm** [comment]
We're using Proxima Nova here since that is the primary typeface we use at DuckDuckGo. As a designer I would always prefer to enforce a consistent design-language throughout a product. In that sense I think it adds a lot to the application, because it makes it ours.

I see that removing the typeface in favor of a default font would make things easier for engineers, but deviating from our design-language would negatively impact our branding, and with that, the people who use the app. I don't necessarily believe that a smoother contribution-process warrants this regression unless the contribution in question adds significant value to the app.

Aren't there alternatives to smoothen the process? can't we somehow fall back to a system typeface when Proxima Nova is not detected?

**Comment by AngeloStavrow** [comment]
That makes a lot of sense to me—I agree that visual consistency across all platforms is really important. I'll look into the system-font-as-fallback option this weekend and send a PR with a better fix. :+1:

**Comment by AngeloStavrow** [comment]
Progress:

I have a custom run script phase that will check the `$CONFIGURATION` and if it's Debug, it basically does nothing. Otherwise, for Release-\* configurations, it should copy the Proxima Nova font files over.

We can then remove the font files from the Copy Bundle Resources phase, and everything ends up rendered in the system font for Debug builds.

However, I'm not entirely sure about the source and destination folders and my script-fu is not strong. :unamused:

Can someone confirm that the files exist in `$(SCROOT)/Resources/proximanova-*.otf`?

The build script is as follows:

```
if [ "${CONFIGURATION}" == Debug ] ; then
    echo "Debug configuration."
    # Basically do nothing here and it'll fall back to the system font.
else
    echo "Release configuration."
    cp -r "${SCRIPT_INPUT_FILE_0}" ${BUILD_PRODUCTS_DIR}/${PRODUCT_NAME}.app
    cp -r "${SCRIPT_INPUT_FILE_1}" ${BUILD_PRODUCTS_DIR}/${PRODUCT_NAME}.app
    cp -r "${SCRIPT_INPUT_FILE_2}" ${BUILD_PRODUCTS_DIR}/${PRODUCT_NAME}.app
    cp -r "${SCRIPT_INPUT_FILE_3}" ${BUILD_PRODUCTS_DIR}/${PRODUCT_NAME}.app
fi
```
